src/coreclr/vm/appdomain.hpp

```diff
@@ -2237,13 +2244,14 @@ class AppDomain : public BaseDomain
     CrstExplicitInit m_nativeImageLoadCrst;
     MapSHash<LPCUTF8, PTR_NativeImage, NativeImageIndexTraits> m_nativeImageMap;
 
+#if defined(FEATURE_COMINTEROP) || defined(FEATURE_COMWRAPPERS)
```

Comment:
Same comment about the `#ifdef` logic - do we need the or version?

Reply:
Same comment as previous